.lintstagedrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   '**/*.{ts,tsx,js,jsx,cjs,mjs}': [
-    'rome format --write',
+    'biome format --write --no-errors-on-unmatched',
     'ts-standard --fix',
     'eslint --cache --fix'
   ],
-  '!(.vscode/**)**/*.json': ['rome format --write'],
+  '!(.vscode/**)**/*.json': ['biome format --write --no-errors-on-unmatched'],
```

Comment:
What kind of formatting issue(s) running biome without `--no-errors-on-unmatched` option trigger?

Reply:
Disclaimer, I never used lint staged.
That option makes sure Biome doesn't error if no files are processed.

Reply:
That should not be needed then. 

Hope you don't mind, I've created branch that I can edit from your PR to fix the issues left: https://github.com/poolifier/poolifier/pull/1129